src/util/fetch/fetch.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { createAbortableFetch } from './fetch'
+import fetchMock from 'jest-fetch-mock'
+
+describe('fetch utilities', () => {
+  beforeEach(() => {
+    fetchMock.resetMocks()
+  })
+  describe('createAbortableFetch', () => {
+    test('aborts previous request', async () => {
+      fetchMock.mockResponses(
+        [JSON.stringify({}), { status: 200 }],
+        [JSON.stringify({}), { status: 200 }]
```

Comment:
I'm not sure I understand correctly this part: is it supposed to mock each request response? 
if it's the case, I think the first one is cancelled so it can't have the 200 status 🤔

Reply:
Yes it is, and you're right, it would be more appropriate to set a `0` status, or no status. I updated the test.